jenkins-jobs/job-dsl/downstream_rhel_prepare.groovy

```diff
@@ -0,0 +1,32 @@
+pipelineJob('downstream-rhel-prepare-prepare-job') {
```

Comment:
use ``rhel-downstram-prepare-job`` and name the fille ``rhel_downstream_prepare.groovy``